server/ServerFSM.cpp

```diff
@@ -735,6 +741,20 @@ sc::result Idle::react(const Hostless&) {
     std::vector<PlayerSaveGameData> player_save_game_data;
     server.InitializePython();
     server.LoadChatHistory();
+
+    if (load_or_quickstart) {
+        std::string subdir = server.m_galaxy_setup_data.GetGameUID();
```

Comment:
maybe add a comment about auto saves being stored in a directory named for the game's UUID?

Reply:
Added.